src/containers/my-resources/add-categories-modal/AddCategoriesModal.constants.ts

```diff
@@ -4,11 +4,17 @@ export const initialValues = {
   name: ''
 }
 
-export const validations = {
-  name: (value: string) =>
-    emptyField(
-      value,
-      'common.errorMessages.emptyField',
-      textField(2, 35)(value)
+const valueNotIn = (value: string, disallowedValues: string[]) =>
+  disallowedValues.includes(value)
+    ? 'myResourcesPage.categories.categoryAlreadyExistsError'
+    : undefined
+
+export const validations = (disallowedValues: string[]) => ({
+  name: (value: string) => {
+    const trimmedValue = value.trim()
+    return (
+      emptyField(value, undefined, textField(2, 35)(trimmedValue)) ||
```

Comment:
Instead of passing `undefined` to you function, you can refactor your function to expect object as an argument and then just not passing that key on function call. Therefor that key will be undefined

Reply:
Refactored emptyField, made object parameter, changed arguments in places where it is used